<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBoxInput">
      <property name="title">
       <string>Параметры</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <!-- Начальная точка -->
       <item row="0" column="0">
        <widget class="QLabel" name="labelInitialPoint">
         <property name="text">
          <string>Начальная точка:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditInitialPoint">
         <property name="text">
          <string>5.0, 5.0</string>
         </property>
        </widget>
       </item>
       
       <!-- Параметры алгоритма -->
       <item row="1" column="0">
        <widget class="QLabel" name="labelAlpha">
         <property name="text">
          <string>Alpha (отражение):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditAlpha">
         <property name="text">
          <string>1.0</string>
         </property>
        </widget>
       </item>
       
       <!-- Gamma (расширение) -->
<item row="2" column="0">
    <widget class="QLabel" name="labelGamma">
        <property name="text">
            <string>Gamma (расширение):</string>
        </property>
    </widget>
</item>
<item row="2" column="1">
    <widget class="QLineEdit" name="lineEditGamma">
        <property name="text">
            <string>2.0</string>
        </property>
    </widget>
</item>

<!-- Beta (сжатие) -->
<item row="3" column="0">
    <widget class="QLabel" name="labelBeta">
        <property name="text">
            <string>Beta (сжатие):</string>
        </property>
    </widget>
</item>
<item row="3" column="1">
    <widget class="QLineEdit" name="lineEditBeta">
        <property name="text">
            <string>0.5</string>
        </property>
    </widget>
</item>

<!-- Sigma (сокращение) -->
<item row="4" column="0">
    <widget class="QLabel" name="labelSigma">
        <property name="text">
            <string>Sigma (сокращение):</string>
        </property>
    </widget>
</item>
<item row="4" column="1">
    <widget class="QLineEdit" name="lineEditSigma">
        <property name="text">
            <string>0.5</string>
        </property>
    </widget>
</item>

<!-- Шаг (step) -->
<item row="5" column="0">
    <widget class="QLabel" name="labelStep">
        <property name="text">
            <string>Начальный шаг:</string>
        </property>
    </widget>
</item>
<item row="5" column="1">
    <widget class="QLineEdit" name="lineEditStep">
        <property name="text">
            <string>1.0</string>
        </property>
    </widget>
</item>

<!-- Допуск (tolerance) -->
<item row="6" column="0">
    <widget class="QLabel" name="labelTolerance">
        <property name="text">
            <string>Критерий остановки:</string>
        </property>
    </widget>
</item>
<item row="6" column="1">
    <widget class="QLineEdit" name="lineEditTolerance">
        <property name="text">
            <string>1e-6</string>
        </property>
    </widget>
</item>

<!-- Макс. итераций -->
<item row="7" column="0">
    <widget class="QLabel" name="labelMaxIter">
        <property name="text">
            <string>Макс. итераций:</string>
        </property>
    </widget>
</item>
<item row="7" column="1">
    <widget class="QLineEdit" name="lineEditMaxIter">
        <property name="text">
            <string>1000</string>
        </property>
    </widget>
</item>
       
      </layout>
     </widget>
    </item>
    
    <item>
     <widget class="QPushButton" name="pushButtonOptimize">
      <property name="text">
       <string>Оптимизировать</string>
      </property>
     </widget>
    </item>
    
    <item>
     <widget class="QGroupBox" name="groupBoxResults">
      <property name="title">
       <string>Результаты</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="textEditResults">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
</ui>